keras_retinanet/utils/eval.py

```diff
@@ -165,7 +171,7 @@ def evaluate(
         A dict mapping class names to mAP scores.
     """
     # gather all detections and annotations
-    all_detections     = _get_detections(generator, model, score_threshold=score_threshold, max_detections=max_detections, save_path=save_path)
+    all_detections, all_inferences     = _get_detections(generator, model, score_threshold=score_threshold, max_detections=max_detections, save_path=save_path)
```

Comment:
Since this no longer aligns with the code below, could you remove the extra spaces before the `=` sign?